app/models/katello/erratum.rb

```diff
@@ -144,6 +144,30 @@ def module_stream_objects
       return streams.flatten.uniq
     end
 
+    def self.import_for_repository(repository)
+      pulp_ids = []
+      service_class = SmartProxy.pulp_master!.content_service(content_type)
+      fetch_only_ids = !repository.content_view.default? &&
+          !repository.repository_type.unique_content_per_repo
```

Comment:
since this is errata, i think we can assume 'repository.repository_type.unique_content_per_repo' is false, correct?